querybook/server/logic/comment.py

```diff
@@ -0,0 +1,142 @@
+from querybook.server.app.db import with_session
+from models.comment import Comment, CommentReaction, DataTableComment, DataCellComment
+
+
+@with_session
+def get_comment_by_id(comment_id: int, session=None):
+    return Comment.get(id=comment_id, session=session)
+
+
+@with_session
+def get_all_comment_data_dict_by_id(comment_id: int, session=None):
+    comment = Comment.get(id=comment_id, session=session)
+    child_comments = (
+        session.query(Comment).filter(Comment.parent_comment_id == comment_id).all()
```

Comment:
can a comment have multiple child comments? If multiple, how will it be displayed on the UI?

is the purpose of this function to get only the direct children  or include all descendants? 

If only the children, will the frontend make a separate api request for each comment, in which case it seems doens't need to return the child_comments here.

If including all descendants, `filter(Comment.parent_comment_id == comment_id).all()` will not be able to do that, it can only get the children.

Reply:
it's only single level nesting